.flake8

```diff
@@ -0,0 +1,105 @@
+[flake8]
+
+# !!! BRING THE PAIN !!!
+select = A,B,B9,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z
```

Comment:
the default select is pretty good, this seems like a strange list

Reply:
I'm trying to enable all of the things, and not whatever things someone else picked.  Some plugins are not on by default…

In any case, this bit isn't new; it's been in the config for a while.